tests/integ-jobs/test_track_from_training_job.py

```diff
@@ -47,6 +47,7 @@ def wait_for_job(job, sagemaker_client):
         while True:
             response = sagemaker_client.describe_training_job(TrainingJobName=job)
             status = response['TrainingJobStatus']
+            print('Training Job status is {}'.format(status))
```

Comment:
I dont know if we want to change print to loggers, but it should def be part of another task

Reply:
its possible to emit output with pytest using:
```
tox -- --capture=no

Reply:
created a task to track